.dockerignore

```diff
@@ -1,3 +1,6 @@
 *.pyc
 .idea
 .git
+.tox
+docs/
+.cache
```

Comment:
Good point about the differences.

I guess the point with `docs/_build` vs. `docs` is that we can't really exclude `docs` from git (for obvious reasons), whereas `docs/_build` is mentioned in `README.rst` under the instructions on how to build the docs (and because that's what I do on my machine).